bitnami/timescaledb-bitnami-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# We have to use the bitnami configuration variable to add timescaledb to
+# shared preload list, or else it gets overwritten.
+if [ -z "$POSTGRESQL_SHARED_PRELOAD_LIBRARIES" ]
+then
+    POSTGRESQL_SHARED_PRELOAD_LIBRARIES=timescaledb
+else
+    POSTGRESQL_SHARED_PRELOAD_LIBRARIES="$POSTGRESQL_SHARED_PRELOAD_LIBRARIES,timescaledb"
+fi
+export POSTGRESQL_SHARED_PRELOAD_LIBRARIES
```

Comment:
I haven't seen this happening - I was only executing some basic commands;
but I've put together some basic test framework to run basic tests on the created images - I plan to contribute the tests to your branch (they don't succeed on the current main branch)
we could probably extend the test commands to cover for this as well?